colcon.pkg

```diff
@@ -0,0 +1,3 @@
+{
+    "test-dependencies": ["googletest-distribution"]
```

Comment:
I just realised, this package actually doesn't have any tests.  So this is probably not needed.

Reply:
Would it be better to leave it here and further add unit-test? ;)

Reply:
In a following PR I meant. This does not affect the build in itself.
It will just compile the googletest-distribution before this package if cloned in the workspace.
If it is not in the workspace it will still try and succeed to build the package.

Reply:
Adding tests would indeed be the best solution :)  I'm good with leaving it here but then we really should add tests soon.